orca-pipelinetemplate/src/test/resources/gate.yml

```diff
@@ -0,0 +1,150 @@
+#document netflix-conventions
+clusters:
+  netflix-conventions:
+    requires:
+      membership:
```

Comment:
I kind of prefer the list being clobbered, personally. Otherwise, how would you remove an element from the list?